src/ext/java/org/opentripplanner/ext/transmodelapi/model/plan/TripQuery.java

```diff
@@ -432,6 +432,23 @@ public static GraphQLFieldDefinition create(
           .defaultValue(preferences.itineraryFilter().debug())
           .build()
       )
+      .argument(
+        GraphQLArgument
+          .newArgument()
+          .name("relaxCostAtDestination")
```

Comment:
I know it's called that internally but I wonder if it's a good name. What does the "at destination" mean?

I personally would suggest something with "pareto" or "optimality".